src/screens/UserPortal/Events/Events.spec.tsx

```
@@ -1772,19 +1772,19 @@ describe('Testing Events Screen [User Portal]', () => {
-            input.allDay === true &&
-            input.location === 'Recurring Test Location' &&
-            input.isPublic === false &&
-            input.isRegisterable === true &&
-            input.isInviteOnly === true &&
-            typeof input.startAt === 'string' &&
-            typeof input.endAt === 'string' &&
-            input.recurrence &&
-            input.recurrence.frequency === Frequency.WEEKLY &&
-            input.recurrence.interval === 1 &&
-            input.recurrence.byDay?.includes(weekDayByJs[dayOfWeek]),
+          input.description === 'Recurring Test Description' &&
```

ESLint

[error] Insert `··`

(prettier/prettier)